ql/src/java/org/apache/hadoop/hive/ql/lockmgr/DbLockManager.java

```diff
@@ -99,7 +99,8 @@ LockState lock(LockRequest lock, String queryId, boolean isBlocking, List<HiveLo
     MAX_SLEEP = Math.max(15000, conf.getTimeVar(HiveConf.ConfVars.HIVE_LOCK_SLEEP_BETWEEN_RETRIES, TimeUnit.MILLISECONDS));
     int maxNumWaits = Math.max(0, conf.getIntVar(HiveConf.ConfVars.HIVE_LOCK_NUMRETRIES));
     try {
-      LOG.info("Requesting: queryId=" + queryId + " " + lock);
+      LOG.info("Requesting: queryId=" + queryId);
```

Comment:
Maybe little more lengthy message could be nice :)
Just imagine seeing this in the log - what could this mean? :)